yt_dlp/extractor/vrt.py

```diff
@@ -176,156 +189,257 @@ def _real_extract(self, url):
         }
 
 
+def jwt_is_expired(token):
+    return jwt_decode_hs256(token)['exp'] - time.time() < 300
+
+
 class VrtNUIE(VRTBaseIE):
     IE_DESC = 'VRT MAX'
-    _VALID_URL = r'https?://(?:www\.)?vrt\.be/vrtnu/a-z/(?:[^/]+/){2}(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?:www\.)?vrt\.be/(?:vrtnu|vrtmax)/a-z/(?:[^/]+/){2}(?P<id>[^/?#&]+)'
```

Comment:
All `/vrtnu/a-z` links are redirecting to `/vrtmax/a-z`, so maybe we can fully remove support for those URLs as well?